appium/lib/screen-object/accessors/element.rb

```diff
@@ -16,17 +16,14 @@
 
 module ScreenObject
   module AppElements
-
     class Element
-
       attr_reader :locator
 
       def initialize(locator)
         if locator.is_a?(String)
-          warn "#{DateTime.now.strftime("%F %T")} WARN ScreenObject Element [DEPRECATION] Passing the locator as a single string with locator type and value sepaarted by ~ is deprecated and will no longer work in version 2.0.0. Use a hash instead (ex: button(:login, id: 'button_id') lib/screen-object/accessors/element.rb:#{__LINE__}"
```

Comment:
This warning was intentional and only to show if you are still passing locator type and value as a string separated by ~ which was original implementation of screen-object we forked. I am already handling the new approach in else case where I check if locator is a hash. We should not change the separator for old approach from ~ to : as it not going to work. Please revert this.